geometry/Building.cpp

```diff
@@ -926,10 +925,10 @@ std::vector<Wall>
 Building::SplitWall(const std::shared_ptr<SubRoom> & subroom, const Wall & bigWall) const
 {
     std::vector<Wall> WallPieces;
-#if DEBUG
-    std::cout << subroom->GetSubRoomID() << "collect wall pieces with " << std::endl;
-    bigWall.WriteToErrorLog();
-#endif
+
```

Comment:
If you refactor like this you probably should also add configurable log levels to the cmdline args. Currently the min log level to write out is "trace" i.e. all your debug messages  will end up in the log, always.

Reply:
Thanks for doing so in #526